cpp/tests/neighbors/ann_vamana.cuh

```diff
@@ -137,6 +138,10 @@ class AnnVamanaTest : public ::testing::TestWithParam<AnnVamanaInputs> {
     index_params.visited_size      = ps.visited_size;
     index_params.max_fraction      = ps.max_fraction;
     index_params.reverse_batchsize = ps.reverse_batchsize;
+    // use codebooks with dim=384 to test serialization & quantization code path
+    if (ps.dim == 384)
+      index_params.codebooks = vamana::deserialize_codebooks(
+        std::string(TEST_DATA_DIR) + "neighbors/ann_vamana/quantizer_fbv6_384_64", ps.dim);
```

Comment:
Hardcoding the path to the codebooks like this seems brittle. There should be a way of passing it in the index params or inputs.